frontend/src/metabase/common/components/CollectionPicker/components/CollectionPickerModal.tsx

```diff
@@ -68,15 +79,15 @@ export const CollectionPickerModal = ({
       if (options.hasConfirmButtons) {
         setSelectedItem(item);
       } else if (canSelectItem(item)) {
-        await onChange(item);
+        await handleChange(item);
```

Comment:
:thinking: 

If this `await` (and the one in `handleConfirm`) is correct then `handleChange` is incorrect (it'd need to be `async` and `await onChange(item)`).

But it looks like the `await` is incorrect - let's remove it.
Also [`MoveModal`](https://github.com/metabase/metabase/blob/8f8a3351b46b3439e62d83c937ae0b740dfad7e7/frontend/src/metabase/containers/MoveModal.tsx#L71) and [`MoveTimelineModal`](https://github.com/metabase/metabase/blob/7ce866130efaa1a331932d3c5cbbf4f3559898b3/frontend/src/metabase/timelines/common/components/MoveTimelineModal/MoveTimelineModal.tsx#L36-L38) should not use `async/await` when passing `onChange` to `CollectionPickerModal`.

Reply:
Good catch!